<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistItem</class>
 <widget class="QWidget" name="PlaylistItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="Head">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="Name">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Stratagem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Difficulty">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Difficulty</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="Stratagems">
     <item>
      <layout class="QHBoxLayout" name="Stratagems_1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Stratagems_2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Stratagems_3"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddStratagemButton">
       <property name="text">
        <string>Add Stratagem</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ControlButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Export">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>PlaylistItem</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
